### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by sebor255** [getgauge/gauge-vscode#831] [General]
I would like to ask when to expect a new release which will include the option to configure the gauge in launch.json file?

**Comment by sriv** [comment]
I've marked a pull request to be a release candidate, a new version will be released post merge (https://github.com/getgauge/gauge-vscode/pull/822). Thanks for the bump!

**Comment by BrudiBanani** [comment]
Thanks for the new release. Unfortunately deployment to marketplace did not work.
Really nice to have the option for launch.json configuration in official release now :)

**Reply by sriv** [reply]
yes, I noticed. I'll need to tweak the release scripts and retrigger. Will do that in some time

**Reply by sriv** [reply]
gauge-vscode 0.1.0 should now be available in the marketplace.

**Reply by sebor255** [reply]
Thanks for the release!!!

**Comment by nagamanickamm** [comment]
Could be great if you can add support for property **"justMyCode": false**
